src/test/java/org/junitpioneer/jupiter/TestUtils.java

```diff
@@ -25,11 +26,10 @@
 	static List<Map<String, String>> reportEntries(ExecutionEventRecorder recorder) {
 		return recorder
 				.eventStream()
-				.filter(event -> event.getType().equals(ExecutionEvent.Type.REPORTING_ENTRY_PUBLISHED))
```

Comment:
This line was deleted because it's not needed - the get payload ensures that we get ReportEntry or empty and we filter out empties. Another option is to remove the Optional::isPresent but that's a 'code smell', I think.

Reply:
But do we still get the correct ReportEntries (wondering about the `REPORTING_ENTRY_PUBLISHED`)?

Reply:
Yes, because the only events in the recorder that have a ReportEntry.class type payload are the report entry events.

Reply:
There is only the `REPORTING_ENTRY_PUBLISHED` type in `ExecutionEvent.Type` that's related to report entries, so the filtering does nothing, I believe.